src/ZipkinBundle/Components/Messenger/B3Stamp.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+use Zipkin\Propagation\RemoteSetter;
+use Zipkin\Propagation\Getter;
+use Zipkin\Kind;
+
+class B3Stamp implements StampInterface, RemoteSetter, Getter
```

Comment:
I am afraid we can't coallese the `StampInterface` with the RemoteSetter/Getter as the former is supposed to be a VO while the RemoteSetter/Getter are stateless classes. I think tho, StampInterface is what the `$carrier` would be.

Reply:
Ok. Do you think is better than I create a specific `RemoteSetter/Getter` or would be better to use something already exists like `Zipkin\Propagation\Map`?

Reply:
I think we need a specific RemoteSetter/Getter anyways, mostly because `Zipkin\Propagation\Map` implements `Setter/Getter` but not the remote counterpart.

Reply:
I take a look at `Zipking\Propagation\Map` I don't like the idea to pass an object that implements `\ArrayAccess` as the carrier because in this case, the carrier would be the stamp. WDYT?

Reply:
I am OK with a completely new setter/getter without using Map. Map already does many things (more than I'd like it to do it) but we can't get rid of it because of retrocompatibility matters.

Reply:
Ok. I created the `PropagationStamp` object used to inject traceId to `B3Stamp`. Could you check if is ok?